docs/src/pages/docs/Miscellaneous/issue_codes.mdx

```diff
@@ -144,8 +144,16 @@ that the username is typed correctly and exists in the database.
 ## Issue 1013
 
 ```
+<<<<<<< HEAD
```

Comment:
Fix this merge conflict?

Reply:
thank you for pointing these out, I went through and checked and didn't see them on my local, pushed up and seems like they are gone.